backend/apps/profile_info/services.py

```diff
@@ -0,0 +1,35 @@
+from django.core.exceptions import ValidationError
+from apps.users.models import BaseUser
+
+
+def update_user_profile(user, validated_data):
+  
+    try:
```

Comment:
Again, these are redundant as user_update can cater for all of these (users/services). And services should use @transaction.atomic alongside keyword only arguments and type annotations.